supplementary/alica_ros1/alica_tests/include/test_alica.h

```diff
@@ -292,35 +276,30 @@ class AlicaTestTracingFixture : public AlicaTestFixtureBase
     virtual void SetUp() override
     {
 
-        // determine the path to the test config
-        ros::NodeHandle nh;
+        // Path to test configs set by CMake
         std::string path;
-        nh.param<std::string>("/rootPath", path, ".");
-        ac = std::make_unique<alica::AlicaContext>(alica::AlicaContextParams("nase", path + "/etc/", getRoleSetName(), getMasterPlanName(), stepEngine()));
+#if defined(PLANS)
```

Comment:
I think we can put this in a function called `getConfigPath()` in one of the base classes. Infact I do just this in another PR https://github.com/rapyuta-robotics/alica/pull/429, but it can be done here itself, since we are modifying this piece of code here anyway

Reply:
I've changed in a very similar way you did, but slightly different. See if you agree.

I've read what you've done and it's fine when you use ROS to create a config file to get some params inside a config file since you get this config file path from a `rosparam`. In my case, I got rid of ROS, so I would need to have the path of the config file somewhere and then inside of this config file, get the path of the test folder. This means I would need the path of the config file again somewhere in my .cpp file in a `constexpr` var, and here I prefer the CMake `target_compile_definitions` approach together with how I did the new base class.

Also, it would be an overkill, at least now, to get params from a YAML file to get one path, and to get the YAML file I would need another hardcoded path inside the .h or .cpp file.